File: NetClan_UICLONE/app/src/main/java/com/redeyesncode/netclanexplorer/activity/MainActivity.java
package com.redeyesncode.netclanexplorer.activity;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.view.GravityCompat;

import android.content.Intent;
import android.os.Bundle;
import android.view.Gravity;
import android.view.LayoutInflater;
import android.view.View;

import com.redeyesncode.netclanexplorer.R;
import com.redeyesncode.netclanexplorer.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    // This activity can be made in kotlin as well. I am familiar with kotlin code style as well.
    
    
    private ActivityMainBinding binding;
    
    


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        
        binding = ActivityMainBinding.inflate(LayoutInflater.from(MainActivity.this));
        
                initClicks();
                
        setContentView(binding.getRoot());
    }

    private void initClicks() {
        binding.ivRefine.setOnClickListener(view -> {
            Intent refineActivityIntent = new Intent(MainActivity.this, RefineActivity.class);
            startActivity(refineActivityIntent);
        });
        binding.ivNav.setOnClickListener(view -> {
            binding.mainDrawerLayout.openDrawer(GravityCompat.START);
        });

    }
}

File: AndroidServicesExample/app/src/main/java/com/redeyesncode/androidservices/MainActivity.kt
package com.redeyesncode.androidservices

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.redeyesncode.androidservices.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    lateinit var binding:ActivityMainBinding


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        //Mainly in android there are three kinds of services which  are described below
        /*  1. Foreground Service 2. Background Service 3. Bounded Service                */


        // In the below lines we are stopping and starting the service.
        binding.btnStartService.setOnClickListener {

            startService(Intent(this,MyAndroidService::class.java))
        }

        binding.btnStopService.setOnClickListener {
            stopService(Intent(this,MyAndroidService::class.java))

        }

    }
}